src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.html

```diff
@@ -147,15 +147,19 @@
     </div>
     <div class="col-item">
       <p class="col-text">{{ 'ubs-tariffs.state' | translate }}</p>
-      <div class="select-item">
-        <mat-form-field>
-          <mat-select #stateSelect formControlName="state" (selectionChange)="onSelectState($event)">
-            <mat-option class="option-state" value="all">Усі</mat-option>
-            <mat-option class="option-state" value="Активно">Активно</mat-option>
-            <mat-option class="option-state" value="Неактивно">Неактивно</mat-option>
-            <mat-option class="option-state" value="Незаповнена">Незаповнена</mat-option>
-          </mat-select>
-        </mat-form-field>
+      <div class="input-wrapper">
+        <mat-select
+          #stateSelect
+          class="state-select"
+          formControlName="state"
+          placeholder="{{ 'ubs-tariffs.placeholder-courier' | translate }}"
+          (selectionChange)="onSelectState($event)"
+        >
+          <mat-option class="option-state" value="Все">{{ 'ubs-tariffs.states.all' | translate }}</mat-option>
+          <mat-option class="option-state" value="Активно">{{ 'ubs-tariffs.states.active' | translate }}</mat-option>
+          <mat-option class="option-state" value="Неактивно">{{ 'ubs-tariffs.states.inactive' | translate }}</mat-option>
+          <mat-option class="option-state" value="Незаповнена">{{ 'ubs-tariffs.states.blank' | translate }}</mat-option>
+        </mat-select>
```

Comment:
Could these options be rendered using *ngFor? Also, shouldn't 'value' be in English lang?